test/modules/aws-elb-integration.ts

```diff
@@ -221,15 +221,16 @@ describe('ELB Integration Testing', () => {
     WHERE load_balancer_name = '${lbName}';
   `, (res: any[]) => expect(res.length).toBe(0)));
 
-  it('deletes the security group 1', query(`
+  it('deletes the security groups', query(`
     DELETE FROM security_group
-    WHERE group_name = '${sg1}';
+    WHERE group_name = ANY(array['${sg1}', '${sg2}']);
   `));
 
-  it('deletes the security group 2', query(`
-    DELETE FROM security_group
-    WHERE group_name = '${sg2}';
-  `));
+  it('check load_balancer delete', query(`
+    SELECT *
+    FROM security_group
+    WHERE group_name = ANY(array['${sg1}', '${sg2}']);
```

Comment:
Same

```suggestion
    WHERE group_name IN ('${sg1}', '${sg2}');
```

This is standard SQL syntax and I would be very surprised if it doesn't work (especially since at least one of the `iasql_*` functions uses it ;)